entrypoint.sh

```diff
@@ -382,6 +382,23 @@ function main::output_software_info {
     aws --version
 }
 
+#######################################
+# Deploy ArgoCD applications via kubectl
+# Globals:
+#   None
+# Arguments:
+#   None
+# Outputs:
+#   Writes progress status
+#######################################
+function deploy_argocd_apps {
```

Comment:
I recommend change name to
```suggestion
function argocd::deploy_apps {
```

Also, just merged PR, where functions moved to `./bin/common.sh` file